src/iop/colorzones.c

```diff
@@ -390,7 +390,7 @@ void process(struct dt_iop_module_t *self, dt_dev_pixelpipe_iop_t *piece, const
     memcpy(ovoid, ivoid, roi_out->width * roi_out->height * ch * sizeof(float));
 
 #ifdef _OPENMP
-#pragma omp parallel for default(none) schedule(static) shared(d)
+#pragma omp parallel for default(none) schedule(static) dt_omp_firstprivate(ivoid, roi_in, roi_out, ch, d, display_channel) shared(ovoid)
```

Comment:
This one works for me:

#ifdef _OPENMP
#pragma omp parallel for default(none) \
  dt_omp_firstprivate(ch, ivoid, ovoid, roi_out) \
  shared(d) \
  schedule(static)
#endif

Reply:
done